### DISCUSSION TITLE
Github actions stuck at queue

### DISCUSSION THREAD

**Discussion by AnnAngela** [community/community#50299] [Actions] [answered]
### Select Topic Area

Question

### Body

[My actions run](https://github.com/MoegirlPediaInterfaceAdmins/MoegirlPediaInterfaceCodes/actions/runs/4443175292) has been stuck as queued for long time.

The run is a re-run on a branch once deleted and restored. Timeline: attempt `#0` -> branch delete -> attempt `#1` -> branch restore -> attempt `#2` (current attempt)

**Comment by jge162** [comment, marked as answer]
just cancel and rerun

**Reply by AnnAngela** [reply]
I cannot cancel it, there is no option to do:

![image](https://user-images.githubusercontent.com/9762652/225852141-47ff2443-f46c-4ede-b343-8d9cdaee7eaf.png)

**Reply by jge162** [reply]
When you click on the 3 dots you don’t see option to cancel

**Reply by AnnAngela** [reply]
![image](https://user-images.githubusercontent.com/9762652/225852828-31259ca6-a7bd-428c-b055-eb4bd09d4804.png)

No, no option to cancel

**Reply by jge162** [reply]
![image](https://user-images.githubusercontent.com/31228460/225852940-c09a219d-144b-4c1a-bae1-efcab8d2852d.png)

**Reply by jge162** [reply]
that is not the right location to click must be on the line of the action

**Reply by AnnAngela** [reply]
Ok, I found it ...

**Reply by jge162** [reply]
> ![image](https://user-images.githubusercontent.com/9762652/225852828-31259ca6-a7bd-428c-b055-eb4bd09d4804.png)
> 
> No, no option to cancel

click here 
![image](https://user-images.githubusercontent.com/31228460/225853374-7a8a40bd-f48a-4b49-acd2-19fe17deb6dd.png)

**Reply by jge162** [reply]
> Ok, I found it ...

perfect
